lib/rubocop/ast/node.rb

```diff
@@ -467,14 +451,11 @@ def parenthesized_call?
       end
 
       def chained?
-        return false unless argument?
-
-        receiver, _method_name, *_args = *parent
-        equal?(receiver)
+        parent && parent.send_type? && eql?(parent.receiver)
       end
 
       def argument?
-        parent && parent.send_type?
+        parent && parent.send_type? && parent.arguments.include?(self)
```

Comment:
This method had a bug in it, where it considered all nodes with a `send` parent an argument, but receivers also have a `send` parent.